controllers/imagejob/imagejob_controller.go

```diff
@@ -165,6 +166,16 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 				Name:            givenImage.Name,
 				ImagePullPolicy: givenImage.ImagePullPolicy,
 				Env:             []v1.EnvVar{{Name: "NODE_NAME", ValueFrom: &v1.EnvVarSource{FieldRef: &v1.ObjectFieldSelector{FieldPath: "spec.nodeName"}}}},
+				Resources: v1.ResourceRequirements{
+					Requests: v1.ResourceList{
+						"cpu":    resource.MustParse("10m"),
+						"memory": resource.MustParse("256Mi"),
```

Comment:
how did you come up with these numbers? these seem too high for memory. can you make sure to check `kubectl top pods -n eraser-system`?